packages/amplify-codegen/src/commands/statements.js

```diff
@@ -4,8 +4,8 @@ const Ora = require('ora');
 
 const loadConfig = require('../codegen-config');
 const constants = require('../constants');
-const { ensureIntrospectionSchema, getFrontEndHandler, getAppSyncAPIDetails } = require('../utils');
-const { generate } = require('@aws-amplify/graphql-docs-generator')
+const { ensureIntrospectionSchema, getFrontEndHandler, getAppSyncAPIDetails, loadSchema, isSDLSchema } = require('../utils');
```

Comment:
`isSDLSchema` doesn't appear to be used.

Reply:
This file is subject to change in upcoming PR. I've made some edits to get e2e passing but am going to re-visit this.